test/algorithms.jl

```diff
@@ -766,6 +766,17 @@ end
                   permute(TH, ((vcat(N, 1:(N - 1))...,), (vcat(2N, (N + 1):(2N - 1))...,)))
         end
     end
+
+    # fermionic tests
+    for N in 3:5
+        h = real(c_plusmin() + c_minplus())
+        H = InfiniteMPOHamiltonian([space(h, 1)], (1, 2) => h)
+        H_periodic = periodic_boundary_conditions(H, N)
+        terms = [(i, i + 1) => h for i in 1:(N - 1)]
+        push!(terms, (1, N) => h)
```

Comment:
I do think writing `push!(terms, (1, N) => permute(h, ((2,1), (4,3)))` would make it more clear why this is the correct term, and would allow to generalize this test to other Hamiltonians which would not have the symmetry that `permute(h, ((2,1), (4,3))) == h`

Reply:
#277